### PR TITLE
project-weather-app - Maria, Emma & Martin (Team Zebras) 

### PR DESCRIPTION
**Pull request by MariaPogosjan**
Link to my deployed project:

[https://weatheroracle.netlify.app/] 

My reflections on how this week's project turned out:
It went great, we had a really good team work and learned a lot from each other. Also I love love json. 

Things I'd like to have clarified or explained in more detail:
One thing we had problem with is that we get undefined on days in the forecast for day in mobile version. 
Don't know why and couldn't solve it.

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        sunset.innerHTML = `
+        <p>
+            ${sunsetTime.getHours()}.${sunsetTime.getMinutes()} 
+        </p>`
+    }).catch(err => {
+
+        document.querySelector('body').style.height = '79vh'
+        document.querySelector('.weather').style.display = 'none'
+        document.querySelector('.current-weather-text').style.display = 'none'
+    })
```
Comment (ettem) [inline code/script.js:62]: Good job catching API errors :)

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            </div>
+            <div class="forecast__icon">
+                <img src="https://openweathermap.org/img/wn/${icon}@2x.png" />
+            </div>
+            <div class="forecast__temp">
+                ${day.main.temp.toFixed(0)} &#176C
+            </div>
+        </div>
+        `
+        document.querySelector('html').style.height = '1500px'
```
Comment (ettem) [inline code/script.js:94]: We wonder about the necessity of this height, in mobile version there is a huge area under the footer.

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+    }) 
+}
+
+const getLocation = () => {
+    if (navigator.geolocation) {
+       navigator.geolocation.getCurrentPosition(geographicUrl)
+    } else (
+        alert ("Your browser doesn't support the geolocation feature")
+    )
+}
```
Comment (ettem) [inline code/script.js:109]: Awesome job with this implementation!

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+    } 
+  }
+
+// Function to start 
+start()
+
+//Event Listener
+submit.addEventListener('click', (event) => {
+    event.preventDefault()
+    console.log(selectedCity)
```
Comment (ettem) [inline code/script.js:147]: You could perhaps delete this console.log

File: code/style.css [hunk 5]
```diff
@@ -0,0 +1,210 @@
+/* Global css variables used for colors */
+
+:root {
+  --first: #1b1a1a; 
+  --hot: linear-gradient(30deg, #ffffff, #e68659);
+  --kindaHot: linear-gradient(30deg, #ffffff, #ffdb3a);
+  --lessHot: linear-gradient(30deg, #f1efef, #2b76da);
+  --aLittleCold: linear-gradient(30deg,#f1efef, #2c54d6);
+  --okayItsCold: #2c1dff;
+}
```
Comment (ettem) [inline code/style.css:10]: Good idea using this :root!

**Review by ettem — APPROVED** [review]
We think this project can be approved and think that you did a really great job! Keep up the good work writing clean code that's easy to read!
/Mette & Maidelin

File: code/script.js [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+    document.querySelector('.current-weather-text').style.display = 'block'
+
+    fetch (APIURL_1)
+    .then ((response) => {
+        return response.json()
+    })
+    .then ((weather) => {
+      yourCity.innerHTML = `${weather.name}`
+
+      typeAndTemperature.innerHTML = `${weather.weather[0].main} | ${weather.main.temp.toFixed(0)}&#176C`
```
Comment (KarolCislowski) [inline code/script.js:43]: The instruction said that you need to present 1 decimal place 🤓

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
It is an awesome project! 🌈  Please fix the decimal place issue I pointed in my comment ad it will be a small masterpiece 🦄

File: code/script.js (hunk 6 above)
Comment (MariaPogosjan) [reply code/script.js]: Hi Karol! 

Thank you for your feedback. We have now added one decimal place as you requested. If you have any answer about why mobile version gives undefined on days, we would be grateful 🤓

**Review by KarolCislowski — APPROVED** [review]
Great job! 

What browser do you use on your mobile? Remember that Safari is a new IE... On my note20 with chrome browser, I can see all days normally. 🤓

**Comment by MariaPogosjan** [comment]
Aah okey cool! I see 🙂 Thank you for your help!

*Allt gott, *
*Maria *

----------

Maria Pogosjan
Phone

|   +46 (0)73 69 14 411  <http://sdf/>




Den mån 1 mars 2021 kl 05:12 skrev Karol Cislowski <notifications@github.com

**Pull request closed without merging**